lgc/patch/NggPrimShader.cpp

```diff
@@ -2080,16 +2080,30 @@ void NggPrimShader::buildPrimShaderWithGs(Function *primShader) {
       assert(m_pipelineState->getShaderModes()->getGeometryShaderMode().outputPrimitive ==
              OutputPrimitives::TriangleStrip);
 
-      // NOTE: primData[N] corresponds to the forming vertices <N, N+1, N+2> or <N, N+2, N+1>.
+      // NOTE: primData[N] corresponds to the forming vertices
+      // The vertice indices in the first triangle <N, N+1, N+2>
+      // If provokingVtxLast is 0, the vertice indices in the second triangle is <N, N+2, N+1>, otherwise it is <N+1, N,
```

Comment:
provokingVtxLast is 0
-> provoking vertex is the first one

Reply:
Fixed.